library/ntc_config_command.py

```diff
@@ -141,8 +141,10 @@
     HAS_NETMIKO=False
 
 
+from packaging import version
```

Comment:
```suggestion
try:
    from packaging import version
    HAS_PACKAGING = True
except ImportError:
    HAS_PACKAGING = False
```

Reply:
thanks I will change, test en update according this solution